backend/nodes/properties/inputs/numeric_inputs.py

```diff
@@ -163,9 +162,23 @@ def __init__(
             step=1,
             optional=optional,
             note_expression=note_expression,
-            hide_ends=hide_ends,
             number_type="slider",
         )
+        self.ends = (min_val, max_val) if ends == (None, None) else ends
+
+    def toDict(self):
+        return {
+            "type": self.input_type,
+            "label": self.label,
+            "min": self.minimum,
+            "max": self.maximum,
+            "noteExpression": self.note_expression,
+            "ends": self.ends,
+            "def": self.default,
+            "step": self.step,
+            "hasHandle": True,
```

Comment:
Well, that an assumption. Is that really true right now? Also, will it also be true in the future?

Reply:
it should be customizable yeah, but also it doesn't really matter since at the moment number types don't have any outputs

Reply:
> 

What is the question here?